db/init/11-batch.sql

```diff
@@ -55,17 +55,17 @@ BEGIN
             AND a.flag_active=true
             AND a.id=ANY(indicator_id)
             ORDER BY a.execution_order
-        ) INSERT INTO base.session (status, indicator_id, batch_id, user_group)
-        SELECT 'Pending', indicator.id, batch.id, '' FROM indicator;
+        ) INSERT INTO base.session (status, indicator_id, batch_id, user_group_id)
+        SELECT 'Pending', indicator.id, batch.id, 0 FROM indicator;
```

Comment:
Which user group has id `0`?

Reply:
I added referential integrity rules on user_group_id columns to allow Postgraphile to introspect the relationship with user_group table. The row level security policies are also based on this column. Those two points mean:
- user_group_id columns should not be null to avoid loosing visibility on the data due to RLS
- we need a default user group to preserve referential integrity with user_group table

So when database is initialize, I have added a default user group to be created with Id = 0. However, I realize this user group is not granted to any user. I will make changes to grant it by default to everyone.